first_timers/first_timers.py

```diff
@@ -10,43 +10,39 @@
 MAX_TWEETS_LEN = 280
 
 ellipse = u'…'
-query_string = 'https://api.github.com/search/issues?q=label:"{}"+is:issue+is:open&sort=updated&order=desc'
-queries = [query_string.format('good-first-issue'), query_string.format('good first issue')]
+FIRST_ISSUE_QUERY_URL = 'https://api.github.com/search/issues?q=label:"{}"+is:issue+is:open&sort=updated&order=desc'
 
 
-def humanize_url(api_url):
+
+def humanize_url(api_url: str) -> str:
     """Make an API endpoint to an Human endpoint."""
     match = re.match('https://api.github.com/repos/(.*)/(.*)/issues/([0-9]*)', api_url)
-    if match is None:
-        raise RuntimeError('Format of API URLs has changed: ', api_url)
-
-    user, repo, issue_num = match.group(1), match.group(2), match.group(3)
-    human_url_template = 'https://github.com/{user}/{repo}/issues/{issue_num}'
-
-    return human_url_template.format(user=user, repo=repo, issue_num=issue_num)
+    if not match:
+        raise ValueError(f'Format of API URLs has changed: {api_url}')
+    user, repo, issue_num = match.group(1, 2, 3)
+    
+    return f'https://github.com/{user}/{repo}/issues/{issue_num}'
 
 
-def get_first_timer_issues(days_old=DAYS_OLD):
+def get_first_timer_issues(issue_label: str) -> list[dict]:
     """Fetches the first page of issues with the label first-timers-label
     which are still open.
     """
+    res = requests.get(FIRST_ISSUE_QUERY_URL.format(issue_label))
+    res.raise_for_status()
```

Comment:
I think we want to still keep support for handling 403 HTTP code like in below line of code.
We want to return the items what ever has been processed if we get a 403 HTTP code.

https://github.com/arshadkazmi42/first-issues/blob/master/first_timers/first_timers.py#L36-L38

Can you make changes to support that.

Reply:
I think that with you current code implementation, you get one request for each issue label.
What i suggest is the program will loop through queries in `run.py` and fetched items will be added to `new_issues` list. And if some http error will be raised, you'll still get successfully fetched items.